go/roothash/api/commitment/pool.go

```diff
@@ -267,28 +239,74 @@ func (p *Pool) addOpenExecutorCommitment(
 		return ErrTxnSchedSigInvalid
 	}
 
-	// Check if the header refers to merkle roots in storage.
-	if uint64(len(body.StorageSignatures)) < p.Runtime.Storage.MinWriteReplication {
-		logger.Debug("executor commitment doesn't have enough storage receipts",
-			"node_id", id,
-			"min_write_replication", p.Runtime.Storage.MinWriteReplication,
-			"num_receipts", len(body.StorageSignatures),
-		)
-		return ErrBadStorageReceipts
-	}
-	if err := sv.VerifyCommitteeSignatures(scheduler.KindStorage, body.StorageSignatures); err != nil {
-		logger.Debug("executor commitment has bad storage receipt signers",
-			"node_id", id,
+	// Validate basic.
```

Comment:
Basic checks should be moved before signature verification (e.g., where RAK checks were before).